test/transcoding/check_wo_qualifier.ll

```diff
@@ -26,20 +26,20 @@ target triple = "spir64-unknown-unknown"
 ; Function Attrs: nounwind
 define spir_kernel void @sample_kernel(%opencl.image2d_array_wo_t addrspace(1)* %input) #0 !kernel_arg_addr_space !1 !kernel_arg_access_qual !2 !kernel_arg_type !3 !kernel_arg_base_type !5 !kernel_arg_type_qual !4 {
 entry:
-  %call.tmp1 = call spir_func <2 x i32> @_Z13get_image_dim20ocl_image2d_array_wo_t(%opencl.image2d_array_wo_t addrspace(1)* %input)
+  %call.tmp1 = call spir_func <2 x i32> @_Z13get_image_dim20ocl_image2d_array_wo(%opencl.image2d_array_wo_t addrspace(1)* %input)
```

Comment:
To ward off the inevitable question here:

These function names have an incorrect mangling: `ocl_image2d_array_wo_t` has a length of 22, not 20. So the demangler will not consider this to be a valid mangled name, which causes the test to break. None of the name mangler sources I'm aware of are mangling it incorrectly (note the correct spelling earlier in the file here in the CHECK-LLVM lines), so my guess is that it was incorrectly written by hand.